Makefile

```diff
@@ -676,10 +676,11 @@ THEMISPP_PACKAGE_FILES += $(includedir)/themispp/
 
 JNI_PACKAGE_FILES += $(jnidir)/$(LIBTHEMISJNI_SO)
 
+deb: MODE_PACKAGING = 1
 deb: DESTDIR = $(BIN_PATH)/deb/root
 deb: PREFIX = /usr
-deb: libdir = $(PREFIX)/$(DEB_LIBDIR)
-deb: jnidir = $(PREFIX)/$(DEB_LIBDIR)/jni
+deb: libdir = $(PREFIX)$(DEB_LIBDIR)
+deb: jnidir = $(PREFIX)$(DEB_LIBDIR)/jni
```

Comment:
Hm... It seem _this_ is the cause of duplicate slashes.

How about leaving it the same (to be consistent with RPM) but fix the `DEB_LIBDIR` to not include the leading slash instead? These lines seem to be the only place where it's used.

Reply:
Yep, it makes sense.

Reply:
I looked at the code. Nope. I'd rather modify `RPM_LIBDIR` and add initial slash to be sure that _directories_ have standard format `/<path>`.

What do you think?

Reply:
It's okay if it's consistent.

I'd argue that LIBDIR paths should not start with a slash because they are not absolute paths and PREFIX is usually not empty (as opposed to DESTDIR). But... okay. If it's consistent and localized then it's fine, I guess.